promptsource/promptsource.py

```diff
@@ -55,7 +55,7 @@ def reset_template_state():
 
 def show_jinja(t, width=WIDTH):
     wrap = textwrap.fill(t, width=width, replace_whitespace=False)
-    out = highlight(wrap, DjangoLexer(), HtmlFormatter())
+    out = highlight(wrap, DjangoLexer(), HtmlFormatter(lineseparator="<br>"))
```

Comment:
This is for visual.
Without it, `prompt + X` of some templates in the viewer mode will have different styles per paragraph due to the `\n` vs `<br>` situation.